admin-cli/src/index.ts

```diff
@@ -5,10 +5,17 @@ import * as davl3 from '@daml2ts/davl-v3/lib/edb5e54da44bc80782890de3fc58edb5cc2
 import * as davlUpgradev3v4 from '@daml2ts/davl-upgrade-v3-v4/lib/b31fe1021c80fcd4e0adc3437d24a328f3b721e81c0a158f6c4a94b89cb8ab32/Upgrade';
 import { Argv } from 'yargs'; // Nice docs : http://yargs.js.org/
 
+type Vacation = {
+  fromDate: string;
+  toDate: string;
+  approved: boolean;
+}
+
 type EmployeeInfo = {
   boss: string;
   vacationDays: number;
   acceptProposal: boolean;
+  vacationRequests: Vacation[];
```

Comment:
Let's make this optional. This makes the config file easier to use.
```suggestion
  vacationRequests?: Vacation[];
```